projects/angular-proto-firestore/src/app/app.component.html

```diff
@@ -1,13 +1,14 @@
-<div *ngIf="!book">
-  Loading
-</div>
-<div *ngIf="book">
-  <div>
-    <span>Title: </span>
-    <span>{{ book.getTitle() }}</span>
-  </div>
-  <div>
-    <span>Amount of pages: </span>
-    <span>{{ book.getPages() }}</span>
-  </div>
+<div *ngIf="!storylist">
+    Loading
 </div>
+<div *ngIf="storylist">
+    <div *ngFor="let storylistChild of storylist">
+        ----
+        <div *ngFor="let story of storylistChild.getStoryList()">
+            {{ story.getProject()}} :
+            <div *ngFor="let storyItem of story.getItemList()">
+                <!-- doesn't show anything -->
```

Comment:
It's ok, there's no data inside:
https://i.imgur.com/FwMcB69.jpg

Reply:
Right, right now the desktop client is not uploading any images.
We need to add that..